.github/labeler.yml

```diff
@@ -0,0 +1,273 @@
+ - integration-tests/kafka/**/*
+
+area/kafka-streams:
+ - extensions/kafka-streams/**/*
+ - integration-tests/kafka-streams/**/*
+
+area/kogito:
+ - extensions/kogito/**/*
+ - integration-tests/kogito/**/*
+ - integration-tests/kogito-maven/**/*
+
+area/kotlin:
+ - extensions/kotlin/**/*
+ - integration-tests/kotlin/**/*
+
+area/kubernetes:
+ - extensions/kubernetes/**/*
+ - extensions/kubernetes-client/**/*
+ - integration-tests/kubernetes/**/*
+ - integration-tests/kubernetes-client/**/*
+
+area/logging:
+ - extensions/logging-gelf/**/*
+ - extensions/logging-json/**/*
+ - extensions/logging-sentry/**/*
+ - integration-tests/logging-gelf/**/*
+
+area/maven:
+ - integration-tests/maven/**/*
+
+area/mongodb:
+ - extensions/mongodb-client/**/*
+ - integration-tests/mongodb-client/**/*
+ - integration-tests/mongodb-panache/**/*
+
+area/narayana:
+ - extensions/narayana-jta/**/*
+ - extensions/narayana-stm/**/*
+ - integration-tests/narayana-jta/**/*
+ - integration-tests/narayana-stm/**/*
+
+area/neo4j:
+ - extensions/neo4j/**/*
+ - integration-tests/neo4j/**/*
+
+area/oidc:
+ - extensions/oidc/**/*
+ - integration-tests/oidc/**/*
+ - integration-tests/oidc-code-flow/**/*
+
+area/panache:
+ - extensions/panache/**/*
+
+area/persistence:
+ - integration-tests/jpa/**/*
+ - integration-tests/jpa-derby/**/*
+ - integration-tests/jpa-h2/**/*
+ - integration-tests/jpa-mariadb/**/*
+ - integration-tests/jpa-mssql/**/*
+ - integration-tests/jpa-mysql/**/*
+ - integration-tests/jpa-postgresql/**/*
+ - integration-tests/jpa-without-entity/**/*
+ - integration-tests/common-jpa-entities/**/*
+
+area/qute:
+ - extensions/qute/**/*
+
+area/reactive:
+ - extensions/reactive-mysql-client/**/*
+ - extensions/reactive-pg-client/**/*
+ - extensions/reactive-streams-operators/**/*
+ - integration-tests/reactive-mysql-client/**/*
+ - integration-tests/reactive-pg-client/**/*
+
+area/rest-client:
+ - extensions/rest-client/**/*
+ - extensions/elasticsearch-rest-client/**/*
+
+area/resteasy:
+ - extensions/resteasy/**/*
+ - extensions/resteasy-common/**/*
+ - extensions/resteasy-jackson/**/*
+ - extensions/resteasy-jaxb/**/*
+ - extensions/resteasy-jsonb/**/*
+ - extensions/resteasy-qute/**/*
+ - extensions/resteasy-server-common/**/*
+ - integration-tests/resteasy-jackson/**/*
+ - integration-tests/elytron-resteasy/**/*
+ - integration-tests/virtual-http-resteasy/**/*
+
+area/scala:
+ - extensions/scala/**/*
+ - integration-tests/scala/**/*
+
+area/scheduler:
+ - extensions/scheduler/**/*
+
+area/security:
+ - extensions/security/**/*
+ - extensions/elytron-security/**/*
+ - extensions/elytron-security-common/**/*
+ - extensions/elytron-security-jdbc/**/*
+ - extensions/elytron-security-oauth2/**/*
+ - extensions/elytron-security-properties-file/**/*
+ - integration-tests/elytron-security/**/*
+ - integration-tests/elytron-security-jdbc/**/*
+ - integration-tests/elytron-security-oauth2/**/*
+
+area/smallrye:
+ - extensions/smallrye-context-propagation/**/*
```

Comment:
To me the specs are different worlds. I think JWT fits better in security, openapi could live together with resteasy, reactive-messaging-* in reactive

Reply:
I'm not fully following - can you make a propsed edit here? otherwise we can update it after I push this initial setup.